src/main/java/core/basesyntax/Penguin.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Penguin extends Animal implements Landy, Swimmable {
```

Comment:
```suggestion
public class Penguin extends Bird implements Landy, Swimmable {
```
let's create `Bird` and `Fish` subclasses of `Animal` as well
(we may need to have functions that are specific for all Birds including Penguin, Sparrow, and others)

Reply:
done